### DISCUSSION TITLE
Comparison Sets updating availability

### DISCUSSION THREAD

**Discussion by wala0003** [datafeedr/wordpress-plugins#21] [Ideas]
Could [this concept of updating prices based on the lowest comp set price](https://datafeedrapi.helpscoutdocs.com/article/198-update-woocommerce-product-prices-to-match-comparison-set-prices) be extended to save whether a product is available or not? Looks like external products don't manage stock by default in WooCommerce, but this could be saved as a new attribute.

**Comment by EricBusch** [comment]
Hi

Yes, in that custom code you have access to the `WC_Product` object (`$product`). You can use that to add a new attribute to the product if you want.

Eric
